### PR TITLE
merge 538 test mvp

### PR DESCRIPTION
**Pull request by mariumtapal**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Package: fivethirtyeight
+Package: fivethirtyeight.test
```
Comment (rudeboybert) [inline DESCRIPTION]: shouldn't the `.test` be removed now?

File: DESCRIPTION [hunk 2]
```diff
@@ -40,8 +40,8 @@ Depends:
 License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
-URL: https://github.com/rudeboybert/fivethirtyeight
-BugReports: https://github.com/rudeboybert/fivethirtyeight/issues
+URL: https://github.com/mariumtapal/fivethirtyeight_test
```
Comment (rudeboybert) [inline DESCRIPTION]: same here?

File: DESCRIPTION [hunk 3]
```diff
@@ -69,5 +69,7 @@ Suggests:
     wordcloud,
     corrplot,
     ggraph,
-    igraph
+    igraph,
+    fivethirtyeightdata.test
```
Comment (rudeboybert) [inline DESCRIPTION]: same here?

**Review by rudeboybert — CHANGES_REQUESTED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (mariumtapal) [reply DESCRIPTION]: yes! I need to update these details

File: R/zzz.R [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+hasData <- function(has_data = .pkgenv$has_data) {
+  if (!has_data) {
+    msg <- paste("To use this function, you must have the",
+                 "`fivethirtyeightdata.test` package installed. See the",
+                 "`fivethirtyeightdata.test` package vignette for more details.")
+    msg <- paste(strwrap(msg), collapse="\n")
+    stop(msg)
+  }
+}
```
Comment (rudeboybert) [inline R/zzz.R]:
```suggestion
}

```

**Review by rudeboybert — APPROVED** [review]

**Pull request merged**